code/headers/qik_2s12v10.hpp

```diff
@@ -65,10 +70,6 @@ namespace r2d2::moving_platform {
         /// is full power forward.
         void set_m1_speed(const int8_t &new_speed);
 
-        /// @brief
-        /// Initializes the qik2s12v10 by resetting it and then setting the baud
-        /// rate.
-        void init();
```

Comment:
Why is init deleted? We need this function to start the qik. It is in the classdiagram

Reply:
It's not deleted it's changed his place inside the qik_2s12v10.hpp as second public method